protocols/jvss/jvss.go

```diff
@@ -1,11 +1,17 @@
-// Joint Verification Secret Sharing, based on the Shamir Secret Sharing
-// algorithm.
+// JVSS provides a threshold signing scheme based on Shamir's joint verifiable
+// secret sharing algorithm and Schnorr signatures. The protocl runs in two
+// phases. During the protocol setup a long-term shared secret is establised
+// between all participants. Afterwards, any of the members can request a
+// signature, which triggers the creation of another, short-term shared secret.
+// Each member then sends its partial signature to the requester which finally
+// puts everything together to get the final Schnorr signature. To verify a
+// given Schnorr signature a member still has to be able to access the
+// long-term shared secret from which that particular signature was created.
+
```

Comment:
Remove this newline. Otherwise the generated `godoc` will show an empty package description (verify yourself. by running `godoc -http=:6060` and visiting  http://localhost:6060/pkg/github.com/dedis/cothority/protocols/jvss/ )